.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -144,6 +152,8 @@ jobs:
     - name: Build LLD
       if: steps.cache-llvm-build.outputs.cache-hit != 'true'
       run: |
+        export PATH=/opt/rh/gcc-toolset-13/root/usr/bin:$PATH
+        export LD_LIBRARY_PATH="/opt/rh/gcc-toolset-13/root/usr/lib64:/opt/rh/gcc-toolset-13/root/usr/lib:$LD_LIBRARY_PATH"
```

Comment:
An alternative to repeatedly setting these variables is to set them once in the Github environment.

Reply:
Ah just noticed you tried something similar before. Have you tried the following format (which we also use elsewhere in actions):
```
echo 'PATH=/opt/rh/gcc-toolset-13/root/usr/bin:$PATH' >> $GITHUB_ENV
echo 'LD_LIBRARY_PATH=/opt/rh/gcc-toolset-13/root/usr/lib64:/opt/rh/gcc-toolset-13/root/usr/lib:$LD_LIBRARY_PATH' >> $GITHUB_ENV
```

Note that this has to be done per *job*, since each job is an independent session.

Reply:
Done. I used `$GITHUB_PATH` instead. And I using `LD_LIBRARY_PATH` was not necessary.